.github/workflows/ci.yml

```diff
@@ -13,19 +13,81 @@ permissions:
   contents: read
```

Comment:
This is not where we want this but we want it in `ci_bors.yml`.

Reply:
This is only so I can do test runs while developing.